http.md

```diff
@@ -0,0 +1,32 @@
+# HTTP
+
+HTTP - short for Hypertext Transfer Protocol; The foundation of data communication for the World Wide Web.
+
+## Classification / acceptance questions
+
+### Apprentice aka _I know it exists_
+- I know what problem is solved by HTTP
+
+### Intermediate aka _I've been looking in the network tab_
+- I know what types of content can be transported via HTTP
+- I know the structure of a HTTP request/response
```

Comment:
```suggestion
- I know the structure of an HTTP request/response
```

Reply:
that is interesting: https://www.quora.com/A-http-or-an-http-which-is-the-correct-grammatical-usage/answer/Rob-Kerr-3

Reply:
Huh, I pronounce it en ejdżtitipi, what would be the other way?